src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -79,6 +79,15 @@ - (void)commonInit {
   _scrollView.showsVerticalScrollIndicator = NO;
   _scrollView.showsHorizontalScrollIndicator = NO;
 
+  if ([[UIView class]
+          respondsToSelector:@selector(userInterfaceLayoutDirectionForSemanticContentAttribute:)] &&
+      [self respondsToSelector:@selector(semanticContentAttribute)] &&
+      [UIView
+           userInterfaceLayoutDirectionForSemanticContentAttribute:self.semanticContentAttribute] ==
+           UIUserInterfaceLayoutDirectionRightToLeft) {
+    [self setRTLEnabled:true];
```

Comment:
bool and BOOL are not the same type in 32-bit builds. Use YES instead.

setRTLEnabled:YES